bin/node/src/commands/genesis/mod.rs

```diff
@@ -121,21 +112,19 @@ fn create_accounts(
     fs::create_dir_all(&accounts_path).context("Failed to create accounts directory")?;
 
     let mut final_accounts = Vec::new();
+    let mut faucet_count = 0;
+    let mut rng = ChaCha20Rng::from_seed(rand::random());
 
     for account in accounts {
         // build offchain account data from account inputs
-        let mut account_data = match account {
+        let (mut account_data, name) = match account {
```

Comment:
I also was thinking about it and decided to keep it as is because we create account data but don't store it to genesis block yet. We will store it later after returning from the function. But I have no strong preference about it, and ready to update the comment if you disagree with my point.